test/smartseq2_multisample/pr/test_inputs.json

```diff
@@ -1,8 +1,5 @@
 {
-    "TestMultiSampleSmartSeq2.expected_core_QC_hash": "",
-    "TestMultiSampleSmartSeq2.expected_qc_tables_hash": "",
-    "TestMultiSampleSmartSeq2.expected_gene_matrix_hash": "",
-    "TestMultiSampleSmartSeq2.expected_isoform_matrix_hash": "",
+    "TestMultiSampleSmartSeq2.truth_loom": "gs://hca-dcp-mint-test-data/smartseq2_multisample/pr_test_truth/ss2_multisample_truth_20191212.loom",
```

Comment:
I noticed this (the bucket path) is not consistent with other entries in the file, should it be moved?